packages/react/src/themes/base/components/Slider/sliderStyles.ts

```diff
@@ -0,0 +1,92 @@
+import { SliderVariables } from './sliderVariables'
+import { SliderProps, SliderState } from '../../../../components/Slider/Slider'
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { selectors } from '../../../../styles/selectors'
```

Comment:
Having this in a separate file is confusing for me...

Reply:
previously I needed `selectors` in more files. moved them back to  `sliderStyles.ts`, thanks